lib/pbench/agent/results.py

```diff
@@ -122,45 +176,137 @@ def make_result_tb(self) -> str:
         with mdlog_name.open("w") as fp:
             mdlog.write(fp)
 
+        # FIXME: /bin/cp pbench.log ${pbench_run_name}/
+
```

Comment:
Need to execute that still.

Reply:
Actually, I am thinking this might be a good time to cut the cord on this behavior.

I have never liked the idea of copying this log file into the pbench result tar ball, without truncating it, and we can't do that because we allow the "results move" behaviors to happen while `bench-scripts` are executing.

I'll remove this comment in favor of tracking it via issue #1680.